pkg/ccl/backupccl/targets.go

```diff
@@ -380,33 +381,55 @@ func checkMissingIntroducedSpans(
 			tablesIntroduced[descpb.ID(tableID)] = struct{}{}
 		}
 
+		requiredIntroduction := func(table *descpb.TableDescriptor) error {
+			if _, required := requiredTables[table.GetID()]; !required {
+				return nil
+			}
+
+			if !includeTableSpans(table) {
+				return nil
+			}
+
+			_, inPrevBackup := prevTables[table.GetID()]
+
+			_, introduced := tablesIntroduced[table.GetID()]
+
+			if !inPrevBackup && !introduced {
+				tableError := errors.Newf("table %q cannot be safely restored from this backup."+
+					" This backup is affected by issue #88042, which produced incorrect backups after an IMPORT."+
```

Comment:
hm, I workshopped the error msg a bunch with steven and aditya and we landed on this because we wanted to communicate everything to the user right away. Also, we link the tech advisory and not the issue, since Steven thought it would be best to keep the user away from the noise on github. 

keeping this for now, unless you have strong feelings about this.

Reply:
Sure, if you think it works. maybe try looking at it in SHOW JOBS, a log and/or db console though, since it won't be as nicely wrapped as it is inline here. We don't _typically_ have a convention of the error message laying out everything you can/should do, but rather giving you just enough to find your way to docs/issue/etc. If you look at other errors, they don't look like this one.

Anyway, I don't feel strongly so yeah, carry on.